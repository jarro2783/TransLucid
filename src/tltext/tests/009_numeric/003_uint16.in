%%
uint16"0";;
uint16"1";;
uint16"12";;
uint16"65535";;

uint16"0" + uint16"0";;
uint16"15" + uint16"0";;
uint16"0" + uint16"15";;
uint16"8" + uint16"5";;

uint16"0" - uint16"0";;
uint16"10" - uint16"0";;
uint16"15" - uint16"3";;

uint16"6" * uint16"1";;
uint16"6" * uint16"0";;

uint16"5" / uint16"1";;
uint16"23" / uint16"5";;

uint16"7" % uint16"4";;

uint16"0" < uint16"1";;
uint16"1" < uint16"1";;
uint16"2" < uint16"1";;

uint16"0" <= uint16"1";;
uint16"1" <= uint16"1";;
uint16"2" <= uint16"1";;

uint16"0" > uint16"1";;
uint16"1" > uint16"1";;
uint16"2" > uint16"1";;

uint16"0" >= uint16"1";;
uint16"1" >= uint16"1";;
uint16"2" >= uint16"1";;

uint16"0" == uint16"1";;
uint16"1" == uint16"1";;
uint16"2" == uint16"1";;

uint16"0" != uint16"1";;
uint16"1" != uint16"1";;
uint16"2" != uint16"1";;

uint16"5"!prec;;
uint16"5"!is_signed;;
