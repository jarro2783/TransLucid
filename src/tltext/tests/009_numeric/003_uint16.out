uint16"0"
uint16"1"
uint16"12"
uint16"65535"
uint16"0"
uint16"15"
uint16"15"
uint16"13"
uint16"0"
uint16"10"
uint16"12"
uint16"6"
uint16"0"
uint16"5"
uint16"4"
uint16"3"
true
false
false
true
true
false
false
false
true
false
true
true
false
true
false
true
false
true
16
false
